donut/modules/groups/routes.py

```diff
@@ -116,8 +116,8 @@ def create_pos_holder(pos_id):
     try:
         helpers.create_position_holder(pos_id, user_id, start_date, end_date)
     except Exception as e:
-        flask.current_app.logger.error('Failed to add position holder:')
-        flask.current_app.logger.exception(e)
+        flask.current_app.logger.exception('Failed to add position holder:\n' +
+                                           e)
```

Comment:
Is there a reason for changing this? `logger.exception()` is useful because it actually logs the stack trace, whereas converting the error to a string only preserves the error message. I also don't think it's valid to concatenate a string with an exception object?

Reply:
So I combined them because otherwise you get two emails sent to you lol if y’all dont mind that I can revert it

Reply:
Wait also `Exception info is added to the logging message` -- the stack trace also appended to the end of the message

Reply:
Okay yeah putting them in the same email is very reasonable. I get this error when trying to concatenate a string with an exception, though:
```
Python 3.7.3 (default, Dec 13 2019, 19:58:14)
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> e = Exception('abc')
>>> 'some text:\n' + e
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate str (not "Exception") to str
```

Reply:
orz i'll format them

Reply:
But I think formatting also only keeps the error message?
```
>>> e = Exception('abc')
>>> f'some text:\n{e}'
'some text:\nabc'
```

Reply:
Well, format would be something like what I had in calendars, 
` f'Error while sharing {calendar_names} with {email} by {username}: \n{e}'`. The exception itself is formatted within the message, and then the `flask.current_app.logger.exception` appends the stack trace at the end like so
```
[2020-03-25 15:37:24,951] ERROR in helpers: Calendar error: HiThere
Traceback (most recent call last):
  File "/home/ziyan/donut/donut/modules/calendar/helpers.py", line 183, in sync_data
    raise NameError('HiThere')
NameError: HiThere```

Reply:
We only want the error message for this portion right? For example, in calendars, I formatted like 
``` flask.current_app.logger.exception(
            f'Error while sharing {calendar_names} with {email} by {username}: \n{e}')```, 
which will email us
```
[2020-03-25 15:59:47,304] ERROR in helpers: Error while sharing ASCIT,Other with moziyan414@gmail.com by wingfril:
HiThere
Traceback (most recent call last):
  File "/home/ziyan/donut/donut/modules/calendar/helpers.py", line 375, in share_calendar
    raise NameError('HiThere')
NameError: HiThere
```

The error message itself is formatted and passed in as an argument to `flask.current_app.logger.exception`, and the stack track is appended to the end of that message.